homeassistant/package_constraints.txt

```diff
@@ -42,7 +42,8 @@ pyudev==0.23.2
 pyyaml==6.0
 requests==2.28.1
 scapy==2.5.0
-sqlalchemy==1.4.45
+sqlalchemy==2.0.0
+sqlalchemy[mypy]==2.0.0
```

Comment:
```suggestion
```
This apparently isn't allowed:

DEPRECATION: Constraints are only allowed to take the form of a package name and a version specifier. Other forms were originally permitted as an accident of the implementation, but were undocumented. The new implementation of the resolver no longer supports these forms. A possible replacement is replacing the constraint with a requirement. Discussion can be found at https://github.com/pypa/pip/issues/8210